datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/service_checks.py

```diff
@@ -131,9 +136,29 @@ def service_checks(check, sync):
 
             # check
             check = service_check.get('check')
+            invalid_chars = INVALID_CHAR_RE.findall(check)
+            invalid_seq = INVALID_SEQ_RE.findall(check)
+            invalid_end = INVALID_END_RE.findall(check)
             if not check or not isinstance(check, str):
                 file_failed = True
                 display_queue.append((echo_failure, '  required non-null string: check'))
+            elif invalid_chars or invalid_seq:
```

Comment:
is not checking for `invalid_end` intentional?

Reply:
Oversight on my part. My bad. I'll correct it.

Reply:
What do you think of removing this first `if statement` all together?